app/models/user.rb

```diff
@@ -14,6 +14,14 @@ class User < ActiveRecord::Base
   validates :email, format: {with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/},
             unless: :has_authentications?
 
+  def review_notification
+    User.joins(:cards).where('review_date > current_date').find_each do |user|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.